docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1383,9 +1377,9 @@ quarkus tls lets-encrypt prepare --domain=<domain-dns-name>
 +
 The `prepare` command does the following:
 
-* Creates a `.letsencrypt` folder in your application's root directory
-* Creates a self-signed domain certificate and private key for your application configured in the previous <<lets-encrypt-prerequisites,Let's Encrypt prerequisites>> step to be able to start and accept HTTPS requests
-* Creates a `.env` configuration file in your application's root directory and configures the application to use the self-signed domain certificate and private key (until we get the Let's Encrypt certificate)
+* Creates a `.letsencrypt` folder in your application's root directory.
+* Creates a self-signed domain certificate and private key for your application configured in the previous <<lets-encrypt-prerequisites,Let's Encrypt prerequisites>> step to start and accept HTTPS requests.
+* Creates a `.env` configuration file in the root directory of your application and configures it to use the self-signed domain certificate and private key until the Let's Encrypt certificate is obtained.
```

Comment:
I thought we were avoiding the passive voice, but here you are adding it (`is obtained`), and that makes it look like the certificate obtains itself, while it is the user who has to (actively) obtain it

Reply:
Fine be me. Will send the tweak.